packs/data/feats.db/one-millimeter-punch.json

```diff
@@ -10,7 +10,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>Your punches have incredible force and control. When you damage a target using One-Inch Punch, you can focus your ki to send the foe flying. If you do, the target must attempt a Fortitude save against your class DC.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target is pushed back 5 feet.</p>\n<p><strong>Failure</strong> The target is pushed back 10 feet.</p>\n<p><strong>Critical Failure</strong> The target is pushed back 10 feet for each action you spent on One-Inch Punch.</p>"
+            "value": "<p>Your punches have incredible force and control. When you damage a target using One-Inch Punch, you can focus your ki to send the foe flying. If you do, the target must attempt a @Check[type:fortitude|dc:resolve(@actor.classDCs.monk.dc.value)] save against your class DC.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target is pushed back 5 feet.</p>\n<p><strong>Failure</strong> The target is pushed back 10 feet.</p>\n<p><strong>Critical Failure</strong> The target is pushed back 10 feet for each action you spent on One-Inch Punch.</p>"
```

Comment:
Should this become a roll note if "one-inch-punch-2" or "one-inch-punch-3" is set?

Reply:
Done